### PR TITLE
Update GUI.py

### PR DESCRIPTION
**Pull request by PARTHAN27**

### CODE REVIEW

**Comment by georgerahul24** [comment]
hi

**Comment by PARTHAN27** [comment]
hello

File: GUI.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def password(event):
         password.passgui = e.get()
         password.usergui = e1.get()
 
-        t.destroy()
+
```
Comment (georgerahul24) [inline GUI.py:43]: why did you delete?

**Review by georgerahul24 — CHANGES_REQUESTED** [review]
